source/features/unfinished-comments.tsx

```diff
@@ -29,7 +29,7 @@ function updateDocumentTitle(): void {
 
 	if (document.visibilityState === 'hidden' && hasDraftComments()) {
 		documentTitle = document.title;
-		document.title = '(Draft comment) ' + document.title;
+		document.title = '✏️ (draft) ' + document.title;
```

Comment:
Maybe even just

```suggestion
		document.title = '✏️ Draft - ' + document.title;
```

I'd avoid `Draft:` because what follows isn't the actual content